src/System.Windows.Forms/tests/UnitTests/AccessibleObjects/PropertyGridToolStripButton.PropertyGridToolStripButtonAccessibleObjectTests.cs

```diff
@@ -0,0 +1,174 @@
+            ToolStripButton propertyPagesButton = propertyGrid.TestAccessor().Dynamic._btnViewPropertyPages;
+            AccessibleObject categoryButtonAccessibleObject = toolStripButtons[0].AccessibilityObject;
+            AccessibleObject alphaButtonAccessibleObject = toolStripButtons[1].AccessibilityObject;
+            AccessibleObject propertyPagesButtonAccessibleObject = propertyPagesButton.AccessibilityObject;
+
+            Assert.True(categoryButtonAccessibleObject.IsPatternSupported(UIA.SelectionItemPatternId));
+            Assert.True(alphaButtonAccessibleObject.IsPatternSupported(UIA.SelectionItemPatternId));
+            Assert.False(propertyPagesButtonAccessibleObject.IsPatternSupported(UIA.SelectionItemPatternId));
+        }
+
+        [WinFormsFact]
+        public void PropertyGridToolStripButtonAccessibleObject_GetPropertyValue_ControlType_IsRadioButton()
+        {
+            using PropertyGrid propertyGrid = new PropertyGrid();
+            ToolStripButton[] toolStripButtons = propertyGrid.TestAccessor().Dynamic._viewSortButtons;
+            AccessibleObject accessibleObject = toolStripButtons[0].AccessibilityObject;
+
+            UiaCore.UIA actual = (UiaCore.UIA)accessibleObject.GetPropertyValue(UiaCore.UIA.ControlTypePropertyId);
+
+            Assert.Equal(UiaCore.UIA.RadioButtonControlTypeId, actual);
+        }
+
+        [WinFormsFact]
+        public void PropertyGridToolStripButtonAccessibleObject_InvokeAddToSelection_UpdatesCheckedStateAndPropertySort()
+        {
+            using PropertyGrid propertyGrid = new PropertyGrid();
+            propertyGrid.CreateControl();
+            ToolStripButton[] toolStripButtons = propertyGrid.TestAccessor().Dynamic._viewSortButtons;
+            ToolStripButton categoryButton = toolStripButtons[0];
+            ToolStripButton alphaButton = toolStripButtons[1];
+            AccessibleObject alphaButtonAccessibleObject = alphaButton.AccessibilityObject;
+            AccessibleObject categoryButtonAccessibleObject = categoryButton.AccessibilityObject;
+
+            Assert.True(categoryButton.Checked);
+            Assert.False(alphaButton.Checked);
+            Assert.Equal(PropertySort.CategorizedAlphabetical, propertyGrid.PropertySort);
+
+            alphaButtonAccessibleObject.AddToSelection();
+
+            Assert.False(categoryButton.Checked);
+            Assert.True(alphaButton.Checked);
+            Assert.Equal(PropertySort.Alphabetical, propertyGrid.PropertySort);
+
+            categoryButtonAccessibleObject.AddToSelection();
+
+            Assert.True(categoryButton.Checked);
+            Assert.False(alphaButton.Checked);
+            Assert.Equal(PropertySort.CategorizedAlphabetical, propertyGrid.PropertySort);
+        }
+
+        [WinFormsFact]
+        public void PropertyGridToolStripButtonAccessibleObject_InvokeInvoke_UpdatesCheckedStateAndPropertySort()
```

Comment:
Looks strange: 
```suggestion
        public void PropertyGridToolStripButtonAccessibleObject_Invoke_UpdatesCheckedStateAndPropertySort()
```

I usually add tests using the following pattern: **`[className]`_`[methodName]`_`[whatItDoes]`_`[conditions (not necessary)]`**
So, I think, we don't need to add "Invoke" before methods in tests names.